compliance-monitor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10-alpine
+WORKDIR /code
+COPY . .
```

Comment:
If you move the COPY Statement after the package-installation you can save some build-time, when you rebuild the container locally because docker layer cache will be used.